packages/create-webpack-app/templates/init/default/README.md

```diff
@@ -0,0 +1,15 @@
+# 🚀 Welcome to your new awesome project!
+
+This project has been created using **webpack-cli**, you can now run
+
+```bash
+npm run build
```

Comment:
https://nodejs.org/docs/latest/api/cli.html#--run

Reply:
I am not sure what I am supposed to do here I would love if you can be a bit more specific,
all I can infer is you want me to change the run -> --run 
```suggestion
npm --run build
```
like this?
also I have read the section in the link provided

Reply:
`--run` is not supported in v20 ,seems to be introduced in v22 only